reference/orders.v2.oas2.yml

```diff
@@ -1470,6 +1525,23 @@ components:
                     phone: ''
                     email: janedoe@example.com
                     form_fields: []
+                  consignments:
+                    pickups: []
+                    shipping: 
+                      addresses:
+                        address_1: '125 Lane'
+                        address_2: '108 Lane'
+                        city: Austin
+                        state: Texas
+                        zip: '78108'
+                        country: United States
+                        country_iso2: US
+                        phone: '1234567890'
+                        email: janedoe@example.com
+                        customfields:
+                          length: 0
```

Comment:
I think this should look pretty much the same as this example:
https://github.com/bigcommerce/docs/blob/b31254b9c1a2db844c1274875dd1f8f7f27d0be1/reference/orders.v2.oas2.yml#L5099-L5135